book/04-git-server/sections/git-daemon.asc

```diff
@@ -20,7 +20,37 @@ $ git daemon --reuseaddr --base-path=/srv/git/ /srv/git/
 If you're running a firewall, you'll also need to punch a hole in it at port 9418 on the box you're setting this up on.
 
 You can daemonize this process a number of ways, depending on the operating system you're running.
-On an Ubuntu machine, you can use an Upstart script.
+
+Since `systemd` is the most common init system among modern Linux distributions, you can use it for that purpose.
+Simply place a file in `/etc/systemd/system/git-daemon.service` with these contents:
+
+[source,console]
+----
+[Unit]
+Description=Start Git Daemon
+
+[Service]
+ExecStart=git daemon --reuseaddr --base-path=/srv/git/ /srv/git/
+
+Restart=always
+RestartSec=500ms
+
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=git-daemon
+
+User=git
+Group=git
+
+[Install]
+WantedBy=multi-user.target
+----
+
+You might have noticed that Git daemon is started here with `git` as both group and user. Modify it to fit your needs. Make sure provided user exists on the system.
```

Comment:
Whoops, looks like we have multiple sentences on a single line here. It _really_ helps diff readability to format with each sentence on its own line.